test/run_tests.pl

```diff
@@ -28,7 +28,7 @@
     @tests = @ARGV;
 }
 my $list_mode = scalar(grep /^list$/, @tests) != 0;
-if (grep /^alltests|list$/, @tests) {
+if (grep /^(alltests|list)$/, @tests) {
```

Comment:
Oh, ok...  then this is fine.  I just wasn't sure what you were after.  Now I know.